toto/toto-run.py

```diff
@@ -93,8 +93,8 @@ def main():
   parser = argparse.ArgumentParser(
       description="Executes link command and records metadata",
       usage="toto-verify.py --step-name <unique step name>\n" +
```

Comment:
Although it's not part of this PR. This should be `toto-run.py` and not `toto-verify.py`. Do you want to fix this here, or do we need an extra PR for this, or should we just hot push to develop?